cloud/kubernetes/bring-your-own-certs/client.yaml

```diff
@@ -20,7 +20,7 @@ spec:
   serviceAccountName: cockroachdb
   containers:
   - name: cockroachdb-client
-    image: cockroachdb/cockroach:v23.1.12
+    image: cockroachdb/cockroach:v23.2.2
```

Comment:
How come we're advancing from 23.1.x to 23.2.x?

Reply:
I posted your message in #db-release-cs-team